integrated_channels/api/v1/sap_success_factors/views.py

```diff
@@ -15,3 +18,59 @@ class SAPSuccessFactorsConfigurationViewSet(PermissionRequiredForIntegratedChann
     permission_required = 'enterprise.can_access_admin_dashboard'
 
     configuration_model = SAPSuccessFactorsEnterpriseCustomerConfiguration
+
+
+@api_view(['GET'])
+@permission_classes([permissions.IsAuthenticated])
```

Comment:
Since it's a sensitive and critical logging flow, I'd suggest adding a more specific permission so we can restrict it's access to other users. Only staff users and Auth0 should be able to call this endpoint. We might need a custom permission class here. Thoughts, @zamanafzal ?